ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh

```diff
@@ -78,6 +79,16 @@ else
   fi
   subfix=$(openssl rand -hex 2)
   CLUSTER_NAME=${CLUSTER_NAME:-"$prefix-$subfix"}
+  # For long cluster name enabled, append a 39 chars long random string to cluster name
+  # Max possible prefix length is 9, max possible subfix length is 4, hyppen is used 2 times, random string of length 39
+  # (9 + 4 + 2 + 39 = 54)
+  if [[ "$LONG_CLUSTER_NAME_ENABLED" == "true" ]]; then
+    long_name_prefix=$(head /dev/urandom | tr -dc a-z0-9 | head -c 39)
```

Comment:
done, using `head /dev/urandom | tr -dc a-z0-9 | head -c 39` did not incluse capital letters as the cluster name will not support capital alphabets